packages/vetrina/src/Vetrina/Purchase/NewPurchase.purs

```diff
@@ -46,15 +46,20 @@ type State =
   , productSelection    :: Maybe Product
   , paymentMethod       :: Maybe PaymentMethod
   , showProductContents :: Boolean
+  , invalidPassword     :: Boolean
```

Comment:
If we anticipate other kinds of errors in the future, then some enumerated type would be a better choice than `Boolean`.

Reply:
Also, I wonder if we should call this just `invalidLogin`.

Reply:
In my opinion that's excessive future proofing for something that's going to be a simple code change if we need it later.

Reply:
I'll leave it like this for now